go.mod

```diff
@@ -9,18 +9,18 @@ require (
 	github.com/bluekeyes/templatetree v0.1.0
 	github.com/c2h5oh/datasize v0.0.0-20171227191756-4eba002a5eae
 	github.com/die-net/lrucache v0.0.0-20181227122439-19a39ef22a11
-	github.com/google/go-github/v38 v38.1.0
-	github.com/google/go-querystring v1.0.0
+	github.com/google/go-github/v40 v40.0.0
```

Comment:
For some reason excavator is bailing out trying to upgrade dependencies due to the breaking major version change with go-github, so doing this by hand instead.